package.json

```diff
@@ -15,71 +15,86 @@
     "name": "Andrei Dumitrescu",
     "url": "https://github.com/andreidmt"
   },
-  "main": "src/index.js",
+  "contributors": [
+    {
+      "name": "David Gil",
+      "url": "https://github.com/dgilperez"
+    }
+  ],
   "keywords": [
     "nodejs",
     "api",
     "server",
     "framework"
   ],
-  "engines": {
-    "node": ">=12"
-  },
+  "type": "module",
+  "exports": "./src/index.js",
+  "sideEffects": false,
+  "files": [
+    "/src"
+  ],
   "scripts": {
     "----UTIL": "",
-    "setup": "rm -rf ./node_modules && npm i && npm audit fix",
-    "setup:ci": "npm ci && npm audit fix",
-    "update": "npm-check -u",
+    "prepare": "git config --local core.hooksPath .githooks",
```

Comment:
Including this branch into my project's `package.json` will cause this to execute, and fail, preventing the build.

Full error: 

```
3145 timing command:install Completed in 81242ms
3146 verbose stack Error: git dep preparation failed
3146 verbose stack     at ChildProcess.<anonymous> (/userhomepath/.asdf/installs/nodejs/16.10.0/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:64:27)
3146 verbose stack     at ChildProcess.emit (node:events:390:28)
3146 verbose stack     at maybeClose (node:internal/child_process:1064:16)
3146 verbose stack     at Socket.<anonymous> (node:internal/child_process:450:11)
3146 verbose stack     at Socket.emit (node:events:390:28)
3146 verbose stack     at Pipe.<anonymous> (node:net:672:12)
3147 verbose cwd /userhomepath/src/holarosita/current/rosita_cloud_ts_functions/functions
3148 verbose Darwin 20.6.0
3149 verbose argv "/userhomepath/.asdf/installs/nodejs/16.10.0/bin/node" "/userhomepath/.asdf/installs/nodejs/16.10.0/bin/npm" "i"
3150 verbose node v16.10.0
3151 verbose npm  v7.24.0
3152 error code 128
3153 error git dep preparation failed
3154 error command /userhomepath/.asdf/installs/nodejs/16.10.0/bin/node /userhomepath/.asdf/installs/nodejs/16.10.0/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/userhomepath/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
3155 error > @asd14/blocks@0.0.1 prepare
3155 error > git config --local core.hooksPath .githooks
3156 error npm WARN using --force Recommended protections disabled.
3156 error fatal: --local solo puedo ser usado dentro de un repositorio
3156 error npm ERR! code 128
3156 error npm ERR! path /userhomepath/.npm/_cacache/tmp/git-clonewGDWgU
3156 error npm ERR! command failed
3156 error npm ERR! command sh -c git config --local core.hooksPath .githooks
3156 error
3156 error npm ERR! A complete log of this run can be found in:
3156 error npm ERR!     /userhomepath/.npm/_logs/2022-05-09T19_26_42_299Z-debug.log
```

I suspect this change was intented for local dev and release purposes, not to bubble up to the projects that include this dependency, correct?

Reply:
Yup, I just confirmed removing that `prepare` line the including works (and apparently it builds 🥳 )

Is `prepare` some reserved word with side-effects?

Reply:
Yes, [prepare](https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts) is a special npm script that gets run after doing a `npm i` for example.

In our case, i'm locally configuring git to look at `.githooks` folder as a source for githooks - this way we can commit them in the repo and we dont need other dependencies like [husky](https://github.com/typicode/husky).

But it looks like it's messing up the consuming repo's env. Will take a look.

Reply:
Super handy. 

I just removed the `prepare` step from this PR temporarily, so I can test this branch on a deployed environment easier.